src/WPF/WPF.Viewer/App.xaml.cs

```diff
@@ -7,7 +7,9 @@
 // "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific
 // language governing permissions and limitations under the License.
 
+using ArcGIS.Helpers;
```

Comment:
These are unused in the main repo with commented code. They are already included in the replacement snippet. Maybe good to still include, your call here.

Reply:
I think it's good to remove it as it's not used, if we want to debug the analytics locally we can always add the using statement when we uncomment the analytics code.